src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/utils/functions.ts

```diff
@@ -152,3 +152,23 @@ export function sanitizeFilename(filename: string) {
   // eslint-disable-next-line no-control-regex
   return filename.replace(/[^\x00-\x7F]/g, "");
 }
+
+/**
+ * Scrolls to a given element using specified options for behavior, block, and inline alignment.
+ *
+ * @param {HTMLElement} element - The element to scroll to.
+ * @param {string} behavior - The scroll behavior ('auto', 'smooth').
+ * @param {string} block - The vertical alignment ('start', 'center', 'end', 'nearest').
+ * @param {string} inline - The horizontal alignment ('start', 'center', 'end', 'nearest').
+ */
+export function scrollToElement(element: HTMLElement, behavior: string = "auto", block: string = "start", inline: string = "nearest") {
```

Comment:
Tiny comment we could type these as ScrollBehavior etc. 

Then you also get auto complete if you use the function? 

Could also default it to smooth if that's what we'll use most of the time.

Reply:
Done!